linera-views/src/backends/rocks_db.rs

```diff
@@ -96,7 +91,7 @@ fn check_key_size(key: &[u8]) -> Result<(), RocksDbStoreInternalError> {
 #[derive(Clone)]
 struct RocksDbStoreExecutor {
     db: Arc<DB>,
-    root_key: Vec<u8>,
+    start_key: Vec<u8>,
```

Comment:
what is a `start_key`? Seems like we're missing field documentation here.

Reply:
It is a fact that what is used externally as `root_key` does not translate exactly like that in the specific code. Building it everytime it is needed would cost a little, which is why it is better to store the key that is going to be used.

We need to add something and it varies according to the context. This happens for `RocksDb`, `DynamoDb`, `IndexDb`, `LineraStorageService`. Description could be added but there would not be unifying explanation as this is all case specific.

About the name, I thought `effective_root_key`, `extended_root_key`. But they are quite long. Which is why I liked `start_key` which is short and reasonable in what it expresses.